<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_AS_keys_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字字符设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动匹配的Zep地址设置：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AS_save">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AS_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_keys_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AS_address8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>401</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
